pxf/pxf-service/src/main/java/org/apache/hawq/pxf/service/ReadVectorizedBridge.java

```diff
@@ -0,0 +1,126 @@
+package org.apache.hawq.pxf.service;
```

Comment:
ReadVectorizedBridge looks very similar to ReadBridge except for getNext() function. Please refactor both classes to avoid duplication